CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## V4.0.2
 
-- Change title and description types from `string` to `ReactNode`.
+- Updated several prop types to `ReactNode` to support wider range of input values.
```

Comment:
I updated this to be "Change title and description types from `string` to `ReactNode` for `<EmptyState>`." In the PR for the info list item third line of text. Which one do you want to go with?

Reply:
Nevermind, that was a pre-emptive comment, I didn't look at the rest of your PR...Your changelog update makes sense so just ignore that last comment.